src/main/java/seedu/address/logic/commands/EligiblePeopleCommand.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonEligibleForPolicyPredicate;
+import seedu.address.model.policy.Policy;
+
+/**
+ * Finds and lists all persons in address book eligible for a given policy.
+ */
+
+public class EligiblePeopleCommand extends Command {
```

Comment:
EligiblePeople does not return the correct output.
Steps to reproduce bug:
1. addcriteria 1 cr/driving license
2. addtag 1 t/driving license
3. addtag 2 t/driving license
4. eligiblepeople 1 (returns 0 people listed)